### PR TITLE
API 6611 loma linda its part 1

### PR DESCRIPTION
**Pull request by bschofield-va**
- Latest version of parent
- working test with hard coded ICNs/Vista
- Support for configurable Vista ICNs/Site combinations used for connectivity IT tests. Support for secret system properties to specified as a file.
- vista-fhir-query-tests/config/.gitignore for system properties
- Testing notes updates in README
- local-docker-image utility this will use vista-fhir-query-tests/config/secrets.conf

### CODE REVIEW

**Comment by bschofield-va** [comment]
# DO NOT MERGE UNTIL DU IS MERGED FIRST

File: vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/SystemDefinitions.java [hunk 1]
```diff
@@ -24,6 +37,32 @@ private static SystemDefinition lab() {
...
+    try (var in = new FileInputStream(secrets)) {
+      properties.load(in);
+      properties
+          .stringPropertyNames()
+          .forEach(p -> System.setProperty(p, properties.getProperty(p)));
+      log.info("Loaded {} secrets", properties.stringPropertyNames().size());
+    } catch (IOException e) {
+      log.warn("No secrets loaded: {}", e.getMessage());
+    }
+    LOADED_SECRETS = true;
```
Comment (jhulbert-va) [inline vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/SystemDefinitions.java:63]: If we didn't load the secrets, why set to true?

Comment (eclendenning-va) [reply vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/SystemDefinitions.java:63]: if the try is successful, we have loaded the secrets

**Review by jhulbert-va — DISMISSED** [review]

**Review by eclendenning-va — APPROVED** [review]

File: vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/TestIds.java [hunk 2]
```diff
@@ -1,14 +1,82 @@
...
 public final class TestIds {
   @NonNull String observationVitalSign;
+
   @NonNull String observationLaboratory;
+
   @NonNull String patient;
+
   @Builder.Default String unknown = "5555555555555";
+
+  @Default @NonNull List<IcnAtSites> patientSites = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (bschofield-va) [reply vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/TestIds.java:26]: what does this mean?

**Review by eclendenning-va — APPROVED** [review]

File: vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/TestIds.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (jhulbert-va) [reply vista-fhir-query-tests/src/main/java/gov/va/api/health/vistafhirquery/tests/TestIds.java:26]: From the imports, it looks like `@Default` is `@Builder.Default`. This line uses the former, the line above it uses the latter.

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**